app/build.gradle

```
@@ -45,4 +45,8 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+
+    // glide
+    implementation 'com.github.bumptech.glide:glide:4.13.0'
+    annotationProcessor 'com.github.bumptech.glide:compiler:4.13.0'
```

`annotationProcessor`의 경우 자바식 어노테이션 프로세서입니당..!

```suggestion
    kapt 'com.github.bumptech.glide:compiler:4.13.0'
```

로 바꿔주시면 좋을 것 같아요 :)